app/controllers/citizens/savings_and_investments_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Citizens
+  class SavingsAndInvestmentsController < BaseController
+    before_action :bank_accounts, :back_link
+
+    def show
+      @form = SavingsAmounts::SavingsAmountsForm.new(current_params)
```

Comment:
I think if you pass the `savings_amount` in as model, you don't have to do the more complicated `current_params` construction:
```ruby
SavingsAmounts::SavingsAmountsForm.new(model: savings_amount)
```

Reply:
I tried and it doesn't throw any error but it doesn't prepopulate the attributes with the values already present in the `savings_amount `

Reply:
Hmm. I'll have to play - but away from this PR. The `current_params` pattern is used everywhere else so is fine here too.